core/commands/get.go

```diff
@@ -159,8 +181,7 @@ func (gw *getWriter) writeArchive(r io.Reader, fpath string) error {
 	defer file.Close()
 
 	fmt.Fprintf(gw.Out, "Saving archive to %s\n", fpath)
-	bar, barR := progressBarForReader(gw.Err, r)
-	bar.Start()
```

Comment:
we should keep the Start with the Finish so it's symmetric calls. less error prone. think of these three lines like:

```go
lk := sync.Lock{}
lk.Lock()
defer lk.Unlock()
```